src/main/resources/templates/gardens/view.html

```diff
@@ -6,35 +6,41 @@
 
 <body class="d-flex flex-column min-vh-100">
 
-    <div th:replace="fragments :: page-header"></div>
+    <nav th:replace="fragments :: page-header"></nav>
 
     <div style="max-width: 85%; " class="container body-content">
         <div class = "row justify-content-center">
-            <h1 th:text="${garden.name}"></h1>
+            <h1 class="garden-title" th:text="${garden.name}"></h1>
         </div>
-        <div class = "row grid-row individual-content">
-            <div class="col" >
+        <div class = "row">
+            <div class="col-md-6 col-sm-10" style="text-align:center;">
                 <img class = "individual-image" th:src="@{${'/user/garden/list/' + garden.id}}" alt = "Water Garden Photo"/>
             </div>
-            <div class="col-8">
-                <p style="margin-bottom: 0"><strong>About our garden...</strong></p>
+            <div class="col-md-6 col-sm-10">
```

Comment:
changed setup here for responsiveness so that picture adjusts to above the text rather than staying next to it

Reply:
nice! that was irritating me :)